src/astronomer/flask_appbuilder/security.py

```diff
@@ -186,6 +186,16 @@ def before_request(self):
             self.get_session.commit()
             if not login_user(user):
                 raise RuntimeError("Error logging user in!")
+            role_names = []
+            for role in user.roles:
+                role_names.append(role.name)
+            session["roles"] = role_names
```

Comment:
We've already got the role names handy, and there could be other _custom_ roles on user.roles that we (by design) want to ignore. We only mamange the membership of the four built in roles (Admin, Editor, User, Viewer)

```suggestion
            session["roles"] = claims['roles']
```